docs/en/models/yolov9.md

```diff
@@ -189,12 +189,14 @@ We would like to acknowledge the YOLOv9 authors for their significant contributi
     === "BibTeX"
 
         ```bibtex
+        {% raw %}
```

Comment:
Same as YOLOv7 comment. Why here and not elsewhere?

Reply:
same as above